.travis.yml

```diff
@@ -3,7 +3,7 @@ sudo: false
 script: "./gradlew check integrationTest"
 
 jdk:
-  - openjdk6
+  - openjdk7
```

Comment:
So openjdk 7 is required for findbugs? I guess this is a reasonable upgrade, do we think there are any risks of compiling/running under a newer jdk?